ocaml/testsuite/tests/templates/basic/bad_instance_arg_name_not_found.reference

```diff
@@ -0,0 +1,3 @@
+File "bad_instance_arg_name_not_found.ml", line 1:
+Error: The module "Monoid_utils" is given argument "List_monoid"
+       for parameter "Banana". Valid parameters for "Monoid_utils": "Monoid"
```

Comment:
Oh well, why say "invalid" "valid" when we can just say "not"?

> Error: module "Monoid_utils" does not have parameter `Banana`.
Hint: it has parameters: "Monoid"

Reply:
Hmm ... I do think it's clearer to spell out the module again rather than say “it” (the other hints also repeat the name of the module) but yeah, the word _valid_ wasn't doing anything. How about:

> Error: The module "Monoid_utils" has no parameter "Banana".
> Hint: Parameters for "Monoid_utils": "Monoid"

Reply:
Yeah I agree. But I want to nit-pick again and say "parameter of monadic_utils" instead of "parameter for monadic_utils":
> Error: "Banana" is not a parameter of the module "Monoid_utils"
Hint: Parameters of "Monoid_utils": "Monoid"

Reply:
Pushed this version